tests/resources/Util.robot

```diff
@@ -11,30 +11,30 @@ Library  DateTime
 ${bin-dir}  /drone/src/github.com/vmware/vic/bin
 
 *** Keywords ***
-Install VIC Appliance To Test Server
+Set Test Environment Variables
     [Arguments]  ${certs}=${false}  ${vol}=default
```

Comment:
I am not a big fan of double layering the default options here... under the typical scenario it gets set correctly by the defaults from the intall vic appliance keyword, but doubling them up here, causes duplication of code maintenance. Can we remove this and just put them in your test cases?